src/Magento/FunctionalTestingFramework/Test/Util/ActionMergeUtil.php

```diff
@@ -29,6 +29,8 @@ class ActionMergeUtil
     const DEFAULT_SKIP_ON_ORDER = 'before';
     const DEFAULT_SKIP_OFF_ORDER = 'after';
     const DEFAULT_WAIT_ORDER = 'after';
+    const APPROVED_ACTIONS = ['fillField', 'magentoCLI', 'field'];
+    const SECRET_MAPPING = ['fillField' => 'fillSecretField'];
```

Comment:
Testing discovered we need to have a reference to `magentoCLISecret` for feature to work fully.

Reply:
Additionally, you will need to add a fallthrough for `magentoCLISecret` in `TestGenerator`

Reply:
Added the magentoCLISecret reference.